bundles/org.openhab.binding.androiddebugbridge/src/main/java/org/openhab/binding/androiddebugbridge/internal/AndroidDebugBridgeDevice.java

```diff
@@ -374,6 +380,225 @@ public void powerOffDevice()
         }
     }
 
+    public void startIntent(String command) throws NumberFormatException, AndroidDebugBridgeDeviceException,
+            ExecutionException, InterruptedException, TimeoutException {
+        String[] commandParts = command.split("\\|\\|");
+        String targetPackage = commandParts[0];
+        var targetPackageParts = targetPackage.split("/");
+        var packageName = targetPackageParts[0];
+        var activityName = targetPackageParts[1];
```

Comment:
Possible `ArrayIndexOutOfBoundsException` here?

Reply:
That's right, thank you!
Should be fixed with the new changes.